src/Path/Problem.hs

```diff
@@ -63,9 +63,9 @@ prettyProblem go ctx = \case
     let t' = withPrec 1 (go ctx t)
         n  = prettyMeta (prettyVar (length ctx))
         b' = withPrec 0 (go (VS n ctx) (fromScopeFin b))
-    in prec 0 (magenta (pretty "∃") <+> pretty (n ::: t') </> magenta dot <+> b')
+    in prec 0 (group (vsep [magenta (pretty "∃") <+> pretty (n ::: t'), magenta dot <+> b']))
```

Comment:
`</>` doesn’t exist in `prettyprinter`. We could have defined our own, but `group` + `vsep` is a simple enough replacement.